e2e-testing/cypress/functions/LoginFunctions.js

```diff
@@ -60,8 +59,8 @@ class LoginFunctions {
   }
 
   verify_login() {
-    cy.get(LoginLocators.loginButton).should('not.exist');
-    cy.get(LoginLocators.accountBtn).should('exist');
+    cy.get(LoginLocators.loginButton).should('exist');
```

Comment:
Actually, I thought that on verification of logging in, it should be the other way around.
Also, I did this because when I was running it on my local machine, there were inconsistencies when I ran it several different times.